docs/changes.rst

```diff
@@ -3,6 +3,10 @@
 Changelog
 ---------
 
+v1.0.2
+~~~~~~
+- Added sample rate into jams instantiation
```

Comment:
"Added sample rate into jams instantiation" --> "Store sample rate in output JAMS inside the scaper sandbox"